### PR TITLE
feat(firestore-bigquery-export): update the import script to write path_params

### PR DESCRIPTION
**Pull request by yamankatby**
fixes: #947

> Taking the branch from #979 consider merging it first

### CODE REVIEW

File: firestore-bigquery-export/scripts/import/src/config.ts [hunk 1]
```diff
@@ -197,3 +217,22 @@ export async function parseConfig(): Promise<CliConfig> {
...
+ * @param text - eg, regions/asia/countries
+ *
+ * @return - eg, { regionId: "asia" }
+ */
+export const resolveWildcardIds = (template: string, text: string) => {
+  const textSegments = text.split("/");
+  return template
+    .split("/")
+    .reduce((previousValue, currentValue, currentIndex) => {
+      if (currentValue.startsWith("{") && currentValue.endsWith("}")) {
```
Comment (dackers86) [inline firestore-bigquery-export/scripts/import/src/config.ts:233]: Is there are a reason for this if statement? Could this be `[oddvalue] = evenValue`?

Comment (yamankatby) [reply firestore-bigquery-export/scripts/import/src/config.ts:233]: I'm not sure what you mean?

Comment (dackers86) [reply firestore-bigquery-export/scripts/import/src/config.ts:213]: Previously we had the values in the form of `path_params` but I see now we don't have those here! Approving

**Comment by dackers86** [comment]
@yamankatby Waiting on CI updates before approving.

**Review by dackers86 — APPROVED** [review]

**Pull request merged**